rpgame/src/main/java/pk/rpgame/controller/GameEngine.java

```diff
@@ -3,6 +3,8 @@
 import pk.rpgame.model.LevelMap;
 import pk.rpgame.model.Room;
 import pk.rpgame.model.builder.FirstLevelBuilder;
+import pk.rpgame.model.living.Hero;
+import pk.rpgame.view.ExplorationView;
```

Comment:
te nowe importy są później używane? jeśli nie to chyba nie ma po co ich tu dodawać